e2e/content-services/metadata/metadata-smoke-tests.e2e.ts

```diff
@@ -120,26 +120,12 @@ describe('Metadata component', () => {
             await expect(await metadataViewPage.getSize()).toEqual(pngFileModel.getContent().getSizeInBytes());
 
             await metadataViewPage.editIconIsDisplayed();
-            await metadataViewPage.informationButtonIsDisplayed();
-            await expect(await metadataViewPage.getInformationButtonText()).toEqual(METADATA.MORE_INFO_BUTTON);
```

Comment:
looks like getInformationButtonText function is no more used in ADF. Can we remove that function completely?